### ISSUE TITLE
The example doesn't add up.

### ISSUE THREAD

**Issue by typeless** [kniren/gota#30]
The function application example in the README:
```
mean := func(s series.Series) series.Series {
    floats := s.Float()
    sum := 0.0
    for _, f := range floats {
        sum += f
    }
    return series.Floats(sum / float64(len(floats)))
}
df.Cbind(mean)
df.Rbind(mean)
```

`CBind` and `RBind` seem to receive a `DataFrame` value rather than a function value. (?)

**Comment by kniren** [comment]
You are right, that is a typo, it should be `df.Capply(mean)` and `df.Rapply(mean)`

**Comment by typeless** [comment]
I don't quite understand how `Rapply` works despite having read the documentation and looked into the code. Are the elements of the row processed by `Rapply` necessarily the same type?

**Comment by kniren** [comment]
When creating a `series.Series` for a given type all the elements will be converted to said type. In case of `Rapply` each row is first casted to the appropriate type with priority: `String -> Float -> Int -> Bool`. What this means is that if you have a column of type `String` and the rest of the columns are of type `Int`, the row will be casted to a String type. This casted row is then passed to the function to apply.

**Comment by kniren** [comment]
Fixed on #40
